### PR TITLE
Add PHPCS config

### PR DESCRIPTION
**Pull request by manicki**

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	<rule ref="vendor/wikibase/wikibase-codesniffer/Wikibase" />
+
+	<rule ref="Generic.Files.LineLength">
+		<properties>
+			<property name="lineLimit" value="120" />
+		</properties>
+	</rule>
+
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
+		<exclude name="Generic.Arrays.DisallowLongArraySyntax.Found" />
```
Comment (Ladsgroup) [inline phpcs.xml]: I think we should fix it instead of excluding it, maybe in another PR.

File: phpcs.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<ruleset name="WikibaseDataModelServices">
+	<rule ref="vendor/wikibase/wikibase-codesniffer/Wikibase" />
+
+	<rule ref="Generic.Files.LineLength">
+		<properties>
+			<property name="lineLimit" value="120" />
```
Comment (Ladsgroup) [inline phpcs.xml]: Isn't it easily fixable? if not, nvm but if yes, let's fix it.

Comment (manicki) [inline phpcs.xml]: https://github.com/wmde/WikibaseDataModelSerialization/pull/229 :)

Comment (Ladsgroup) [inline phpcs.xml]: You are extremely fast.

File: phpcs.xml (hunk 1 above)
Comment (manicki) [inline phpcs.xml]: I didn't count how many issues this found but there were several. I've first raised the limit to 120 and there were only two problems so I chopped those down. I can make it use the default (100 chars AFAIK) but then I am going to be completely ignoring comments like "The diff looks horrible" :)

**Comment by Ladsgroup** [comment]
Travis is not happy about "ci" command, otherwise good to merge.

**Comment by manicki** [comment]
oops, fixed

File: phpcs.xml (hunk 1 above)
Comment (Ladsgroup) [inline phpcs.xml]: Let's do it in another PR

**Comment by Ladsgroup** [comment]
Scrutnizer is crazy and we need to fix that too.
